src/test/suites/interface-suite.js

```diff
@@ -186,17 +186,33 @@ suite.declare((lab, variables) => {
       return done();
     });
 
-    lab.test("A null source field throws an error", done => {
+    lab.test("A null source field should map the source to destination", done => {
 
       const map = createSut();
 
-      const throws = function () {
+      const result = map(null).execute({});
```

Comment:
I think you need something tangible as the source object. This test doesn't really prove anything to me.